pkg/reconciler/contour/contour.go

```diff
@@ -116,7 +132,7 @@ func (r *Reconciler) ReconcileKind(ctx context.Context, ing *v1alpha1.Ingress) r
 		// The endpoints ingress is ready, we are good to go!
 		haveEndpointProbe = true
 	} else {
-		desiredChIng := resources.MakeEndpointProbeIngress(ctx, ing)
+		desiredChIng := resources.MakeEndpointProbeIngress(ctx, ing, nil)
```

Comment:
comment on the `nil /*not sure what this does*/`.

Reply:
I was being lazy about creating a names package, so I did that rather than write even more trying to explain my laziness.